Various minor docs improvements (#7626)
cvc5/cvc5: docs/api/api.rst

API Documentation
=================

> Apart from using cvc5 :doc:`as a binary <../binary/binary>`, cvc5 features APIs

Reviewer: ```suggestion
In addition to using cvc5 :doc:`as a binary <../binary/binary>`, cvc5 features APIs
```
